packages/api/src/controllers/sensorController.js

```diff
@@ -30,6 +30,14 @@ const sensorErrors = require('../util/sensorErrors');
 
 const sensorController = {
   async addSensors(req, res) {
+    let hasTimedOut = false;
```

Comment:
Technically this comment is for the line above the one marked:
See [simplify-routing](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1191575576/Pre-integration+peer+review#Backend-code): avoid functions that return constant functions.

Reply:
@smattingly I'm not sure what you are referring to here - as far as I can tell `addSensors` is not returning a constant function?

Reply:
Sorry, you're right. I guess I just entirely misread it.